owner/onDieCache.py

```diff
@@ -0,0 +1,61 @@
+#This script copies OnDie ECDSA artifacts to a specified local directory.
```

Comment:
Is this same as scripts/onDieCache.py? If so, we won't need this file. We can move the scripts/onDieCache.py file into demo/owner folder.

Reply:
Not an issue when rebased on top of latest master code.

Reply:
the pri/owner version of the script only downloads the crl files while the SCT version of the script downloads both certs and crls. There should only be one version for the pri.